space-concordia-robotics/robotics-prototype
# [#311] Logging function behaving similarly to appendToConsole()

# Assignee Section

## Description
I made my logging functions behave more like Peter's `appendToConsole()` function. `logDebug()` up to `logFatal()`.

Steps to test:
1) Start the GUI with `rosgui` and `startgui`
2) Go to `localhost:5000` in chrome, open the dev console (`F12`)
3) Call the different logging functions (logDebug, logInfo, logWarn, logErr, logFatal) with a message and one or two boolean parameters. Compare with how the `appendToConsole()` method behaves, they should print to the same consoles according to the boolean flags passed, though appendToConsole should not generate any roslogs. Use `rostopic echo /rosout` to monitor roslogs.

closes #311

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
